Source/bmalloc/libpas/src/libpas/pas_mte.h

```diff
@@ -464,6 +449,38 @@ PAS_IGNORE_WARNINGS_END
         b &= ~PAS_MTE_TAG_MASK; \
     } while (0)
 
+// Use these to configure the tagging policy for different sizes. Currently we only
```

Comment:
This whole chunk was just moved upwards from later on in the file, as it's now needed by pas_mte_tag_allocated_region_as_appropriate, a little further down in the file.